### PR TITLE
ESL update

### PR DESCRIPTION
**Pull request by PolinaVolchetskaya**

### CODE REVIEW

**Review by Sisha3342 — APPROVED** [review]

**Review by AlexanderBazukevich — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

File: src/core/base/root.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import {bind, EventUtils, ObserverCallback, ESLMediaRuleList} from '@exadel/esl';
+import {bind, EventUtils, ESLMediaRuleList} from '@exadel/esl';
```
Comment (ala-n) [inline src/core/base/root.ts:1]: Cosmetic: that might be better to use exact paths (or at least make all exact(`@exadel/esl/modules/..`) or global (`@exadel/esl`))

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
   },
   "homepage": "https://github.com/exadel-inc/ui-playground#readme",
   "dependencies": {
-    "@exadel/esl": "^2.8.1",
+    "@exadel/esl": "^3.0.0-beta.14",
```
Comment (ala-n) [inline package.json:60]: Please update to the latest one. There was a couple of more releases.

**Review by ala-n — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (PolinaVolchetskaya) [reply package.json]: thanks, fixed :)

File: src/core/base/root.ts (hunk 1 above)
Comment (PolinaVolchetskaya) [reply src/core/base/root.ts]: so, should it be like now?

Comment (ala-n) [reply src/core/base/root.ts]: Yep, now everything looks fine
Thanks!

**Review by ala-n — APPROVED** [review]

**Pull request merged**